recipes/libsecret/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "libsecret" %}
+{% set version = "0.18.6" %}
+{% set sha256 = "5efbc890ba41a323ffe0599cd260fd12bd8eb62a04aa1bd1b2762575d253d66f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://ftp.gnome.org/pub/gnome/sources/libsecret/0.18/libsecret-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-Disable-tests-which-require-additional-deps.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    #  https://abi-laboratory.pro/tracker/timeline/libsecret/
+    - {{ pin_subpackage('libsecret', max_pin='x.x') }}
```

Comment:
For now, can we address in issue ( https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/88 ).

Reply:
These lines should now be dropped.

Reply:
I am confused now. Why should this line be removed?

Reply:
Because it will be added to global pinnings with PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/89 ).

Reply:
I was confused because I saw the following line in conda-forge-pinning-feedstock:
```
# TODO: remove these when run_exports are added to the packages.
```

That being said, (I didn't know about this a week before now) I do like the idea of having `pin_run_as_build` so that one doesn't have to rely on the recipe maintainer, but have the pinnings at a central place. 

However, there is one downside to it. What if I really want a version > pin_at_central_location and still want the auto-run-dep inject functionality, and the api compatibility might have changed for a future version of the same package? For example, if they say screw semver, let's break things at patch releases now, for a future version?

Reply:
I think it is a lot easier to stick to the one most common strategy until we are ready to change. Mixing and matching them will make things confusing and harder to maintain. Right now that is the global pinning file. In the future that may change and we can update this accordingly.

Reply:
Whatever happens with conda-forge-pinning, we should not remove the `run_exports` line. We have `run_exports` in most of the packages in `conda-forge-pinning`. Why do you want it dropped?

Reply:
Let's centralize this conversation: https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/102